clang/lib/Headers/avxintrin.h

```diff
@@ -207,6 +207,8 @@ _mm256_div_ps(__m256 __a, __m256 __b)
 /// Compares two 256-bit vectors of [4 x double] and returns the greater
 ///    of each pair of values.
 ///
+///    If either value in a comparison is NaN, returns the value from \a __b.
```

Comment:
Is it worth to mention the ±0 comparison?

Reply:
I don't think so either - we don't need to start explaining general fp comparison behaviour, just any x86/sse quirks.